src/plugins/emoji.js

```diff
@@ -1,1523 +1,1812 @@
+  white_medium_square: 'unicode/25fb',
+  white_small_square: 'unicode/25ab',
+  white_square_button: 'unicode/1f533',
+  wilted_flower: 'unicode/1f940',
+  wind_chime: 'unicode/1f390',
+  wind_face: 'unicode/1f32c',
+  wine_glass: 'unicode/1f377',
+  wink: 'unicode/1f609',
+  wolf: 'unicode/1f43a',
+  woman: 'unicode/1f469',
+  woman_artist: 'unicode/1f469-1f3a8',
+  woman_astronaut: 'unicode/1f469-1f680',
+  woman_cartwheeling: 'unicode/1f938-2640',
+  woman_cook: 'unicode/1f469-1f373',
+  woman_dancing: 'unicode/1f483',
+  woman_facepalming: 'unicode/1f926-2640',
+  woman_factory_worker: 'unicode/1f469-1f3ed',
+  woman_farmer: 'unicode/1f469-1f33e',
+  woman_firefighter: 'unicode/1f469-1f692',
+  woman_health_worker: 'unicode/1f469-2695',
+  woman_in_manual_wheelchair: 'unicode/1f469-1f9bd',
+  woman_in_motorized_wheelchair: 'unicode/1f469-1f9bc',
+  woman_judge: 'unicode/1f469-2696',
+  woman_juggling: 'unicode/1f939-2640',
+  woman_mechanic: 'unicode/1f469-1f527',
+  woman_office_worker: 'unicode/1f469-1f4bc',
+  woman_pilot: 'unicode/1f469-2708',
+  woman_playing_handball: 'unicode/1f93e-2640',
+  woman_playing_water_polo: 'unicode/1f93d-2640',
+  woman_scientist: 'unicode/1f469-1f52c',
+  woman_shrugging: 'unicode/1f937-2640',
+  woman_singer: 'unicode/1f469-1f3a4',
+  woman_student: 'unicode/1f469-1f393',
+  woman_teacher: 'unicode/1f469-1f3eb',
+  woman_technologist: 'unicode/1f469-1f4bb',
+  woman_with_headscarf: 'unicode/1f9d5',
+  woman_with_probing_cane: 'unicode/1f469-1f9af',
+  woman_with_turban: 'unicode/1f473-2640',
+  womans_clothes: 'unicode/1f45a',
+  womans_hat: 'unicode/1f452',
+  women_wrestling: 'unicode/1f93c-2640',
+  womens: 'unicode/1f6ba',
+  woozy_face: 'unicode/1f974',
+  world_map: 'unicode/1f5fa',
+  worried: 'unicode/1f61f',
+  wrench: 'unicode/1f527',
+  wrestling: 'unicode/1f93c',
+  writing_hand: 'unicode/270d',
+  x: 'unicode/274c',
+  yarn: 'unicode/1f9f6',
+  yawning_face: 'unicode/1f971',
+  yellow_circle: 'unicode/1f7e1',
+  yellow_heart: 'unicode/1f49b',
+  yellow_square: 'unicode/1f7e8',
+  yemen: 'unicode/1f1fe-1f1ea',
+  yen: 'unicode/1f4b4',
+  yin_yang: 'unicode/262f',
+  yo_yo: 'unicode/1fa80',
+  yum: 'unicode/1f60b',
+  zambia: 'unicode/1f1ff-1f1f2',
+  zany_face: 'unicode/1f92a',
+  zap: 'unicode/26a1',
+  zebra: 'unicode/1f993',
+  zero: 'unicode/0030-20e3',
+  zimbabwe: 'unicode/1f1ff-1f1fc',
+  zipper_mouth_face: 'unicode/1f910',
+  zombie: 'unicode/1f9df',
+  zombie_man: 'unicode/1f9df-2642',
+  zombie_woman: 'unicode/1f9df-2640',
+  zzz: 'unicode/1f4a4',
+};
 
-// Emoji from All-Github-Emoji-Icons
-// https://github.com/scotch-io/All-Github-Emoji-Icons
+// Emoji from GitHub API
+// https://api.github.com/emojis
 window.emojify = function(match, $1) {
-  return AllGithubEmoji.indexOf($1) === -1
+  return AllGithubEmoji.hasOwnProperty($1) === false
     ? match
     : '<img class="emoji" src="https://github.githubassets.com/images/icons/emoji/' +
-        $1 +
+        AllGithubEmoji[$1] +
```

Comment:
LGTM.

What's the main difference between the two types of links? f.e.

- https://github.githubassets.com/images/icons/emoji/unicode/1f4a4.png
- https://github.githubassets.com/images/icons/emoji/zzz.png

Reply:
The picture presents different sizes, It should be no problem to use as an emoticon.

Reply:
Gotcha. So smaller images, better for in text.